enterprise/wsproxy/wsproxysdk/proxyinternal.go

```diff
@@ -0,0 +1,83 @@
+package wsproxysdk
```

Comment:
I feel like this could go inside of the main `wsproxysdk` file. It's so small anyways I think decomposing it like this makes it more confusing.

Reply:
More methods (registration, healthcheck, ping, etc.) will be added here over time so I'd like to keep it separated.

Reply:
Sure!